index.css

```
@@ -0,0 +1,205 @@
+  --color-dark-muted: #94a3b8;
+
+  --font-sans: "Fredoka", "Outfit", sans-serif;
+  --font-display: "Outfit", sans-serif;
+  --font-mono: "Courier New", monospace;
+
+  --radius-xl: 1rem;
+  --radius-2xl: 1.5rem;
+  --radius-3xl: 2rem;
+
+  --animate-bounce-slow: bounce 3s infinite;
+  --animate-progress-simulate: progressSim 4s ease-out forwards;
```

Stylelint (17.4.0)

[error] Expected "progressSim" to be "progresssim" (value-keyword-case)

(value-keyword-case)